tests/nbody-cp-gradient/input.dat

```diff
@@ -166,7 +166,7 @@ for result in [  #TEST
 
 # compute nbody again with return_total_data=False
 # * note CURRENT ENERGY moved from first to second testing block
-g, wfn = gradient('SCF/STO-3G', molecule=water_trimer, bsse_type='cp', max_nbody=2,
+g, wfn = gradient('SCF/STO-3G', molecule=water_trimer, bsse_type=['cp','nocp'], max_nbody=2,
```

Comment:
is the change in the test still needed now that `return_total_data` defaults to True for grad?

Reply:
Yes, because we're checking with `return_total_data=False` stated explicitly.